mne_qt_browser/_pg_figure.py

```diff
@@ -1697,10 +1697,10 @@ def __init__(self, main, **kwargs):
 class ProjDialog(_BaseDialog):
     """A dialog to toggle projections."""
 
-    def __init__(self, main, *, name):
+    def __init__(self, main, title='Projectors', **kwargs):
```

Comment:
Why do we need the flexibility to adjust the title of that dialog?

Also, in #101 I specifically changed the signature to **not** accept `kwargs` and instead enforce passing keyword-only arguments to ensure better code health. Please revert the change in this line

Reply:
this one also needs to be reverted. We really try to avoid kwargs in mne as it makes the code super hard to document and test.

Reply:
I spent some time looking at the code with @juangpc and I agree **kwargs made the code really hard to read. And it's nested really deep with the backend handling. What used to be "params" in the past is now "kwargs"

Reply:
I see. Well. I'm sorry. I'll fix this.

> Why do we need the flexibility to adjust the title of that dialog?

I don't need it. I changed it back because: 1. coherence. 2. your code when improperly merged with mine, introduced a bug. 
Attending 1) there are 5 or so dialog windows child classes to _BaseDialog. And the way the child classes communicate with it in the other instances is different from what you proposed in your change. See SettingsDialog or HelpDialog, all go in the same toolbar.
Attending 2) The merge with your change introduced a bug in my fork. It might have been me or automatic merge. Don't remember, it was late. This has been solved now. I was surprised by your change, the tests found it. The tests I push here, actually found the bug. So I guess thats a good thing. Anyway it was a bad merge. I'll fix this.

Also, I get a warning when using ```name``` as argument in line 1700: "Shadows name 'name' from greater scope".

I don't know the difference between single star "*" and "**kwargs". This is my first python pr.